app/services/real_performance_tracker.py

```
@@ -0,0 +1,369 @@
+                avg_trade_pnl=Decimal(str(metrics.get('avg_trade_pnl', 0))),
+                total_fees=Decimal(str(metrics.get('total_fees', 0))),
+                is_live=True,
+                data_source='real_trades'
+            )
+
+            db.add(history)
+            await db.commit()
+
+            self.logger.info("✅ Stored performance history", strategy_id=strategy_id)
+
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)